packages/amplify-category-storage/tsconfig.json

```diff
@@ -11,8 +11,8 @@
     "resources",
   ],
   "references": [
-    { "path": "../amplify-cli-core" },
+    {"path": "../amplify-cli-core"},
     { "path": "../amplify-prompts" },
-    { "path": "../amplify-util-import" }
+    {"path": "../amplify-util-import"},
```

Comment:
keep the spaces as they were and remove trailing comma since this is a JSON file

Reply:
Done.